islandora_pathauto.module

```diff
@@ -112,7 +112,8 @@ function islandora_pathauto_create_alias($object, $op) {
  * Implements hook_islandora_object_ingested().
  */
 function islandora_pathauto_islandora_object_ingested($object) {
-  islandora_pathauto_create_alias($object, 'insert');
+    if( !path_load('islandora/object/' . $object -> id) )
```

Comment:
Watch for changes in coding standards.  It looks like Islandora uses explicit braces even for singleton ifs.